lib/net/ssh/service/forward.rb

```diff
@@ -28,6 +28,7 @@ def initialize(session)
       @remote_forwarded_ports = {}
       @local_forwarded_ports = {}
       @agent_forwarded = false
+      @local_forwarded_sockets = {}
```

Comment:
Now properly used. I've also added a test and adjusted the playbook to provision our own sshd, whether that works as expected on travis remains to be seen ^^